src/pages/community/foundation/local-initiative.mdx

```diff
@@ -63,6 +63,40 @@ shall not be referred to as "official", but are still encouraged to follow the
 guidelines that are applicable (in particular, the Code of Conduct and Brand
 Guidelines must still be honoured).
 
+### Local identity
+
+Locals are encouraged to use the name of their area in their Local name and identity.
+The area chosen to be in the Local name should be meaningfully "local". It should
+be culturally identified as a "local area", and at its largest should be a region that you
```

Comment:
I've never understood _why_ this is the case, but apparently [punctuation should be inside of quotation marks](https://www.thesaurus.com/e/grammar/does-punctuation-go-inside-or-outside-quotation-marks/).

```suggestion
The area chosen to be in the Local name should be meaningfully "local." It should
be culturally identified as a "local area," and at its largest should be a region that you
```